sources/Algo/src/GeneratorDefinitionAlgorithm.cpp

```diff
@@ -36,40 +38,60 @@ GeneratorDefinitionAlgorithm::operator()(const NodePtr& node, std::shared_ptr<Al
       dfl::inputs::NetworkManager::NbOfRegulating nbOfRegulatingGenerators = it->second;
       model = ModelType::SIGNALN_INFINITE;
       algoRes->isAtLeastOneGeneratorRegulating = true;
-      if (node_generators.size() == 1 && IsOtherGeneratorConnectedBySwitches(node)) {
+      if (generator.targetV > tfoVoltageLevel_) {
```

Comment:
Should it be targetV or simply the voltage level of the connection bus ?

Reply:
you mean the VNom?

Reply:
Yes, for me it should be the nominal voltage of the bus to which the generator is connected instead of its targetV. It won't change anything as we chose the parameter value accordingly but it seems more consistent